src/mol-repr/volume/util.ts

```diff
@@ -351,3 +352,46 @@ export function createSegmentTexture2d(volume: Volume, set: number[], bbox: Box3
 
     return textureImage;
 }
+
+/**
+ * Create a new volume that is wrapped by one cell in all dimensions.
+ * Reuses the original volume grid data with new data accessors.
+ */
+export function createWrappedVolume(volume: Volume): Volume {
```

Comment:
Why not add this at the `Tensor` level with extra `periodic` option?

Could then bake this into the base getters directly.

Reply:
at some point, yes, this is a simple method only intended for isosurfaces